<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZEqualizerDialog</class>
 <widget class="QDialog" name="ZEqualizerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Channels</string>
       </property>
       <property name="buddy">
        <cstring>spinChannels</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinChannels">
       <property name="specialValueText">
        <string>default</string>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Controls file</string>
       </property>
       <property name="buddy">
        <cstring>editControls</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editControls">
       <property name="toolTip">
        <string>Leave empty for default value - $HOME/.alsaequal.bin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonControlsBrowse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>LADSPA plugin:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPlugin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPluginSelect">
       <property name="text">
        <string>Select...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPluginClear">
       <property name="text">
        <string>Use default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinChannels</tabstop>
  <tabstop>editControls</tabstop>
  <tabstop>buttonControlsBrowse</tabstop>
  <tabstop>buttonPluginSelect</tabstop>
  <tabstop>buttonPluginClear</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ZEqualizerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ZEqualizerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
